sql/moz-fx-data-shared-prod/looker_forecasts_cache/view.sql

```diff
@@ -0,0 +1,57 @@
+CREATE OR REPLACE VIEW
+  `moz-fx-data-shared-prod.telemetry.looker_forecasts_cache`
```

Comment:
Is this view only intended to be used by Looker? If so, I'd say put it in `telemetry_derived` so that it's not user-facing for other contexts.